src/applications/disability-benefits/526EZ/config/form.js

```diff
@@ -545,6 +545,7 @@ const formConfig = {
                 type: 'array',
                 items: {
                   type: 'object',
+                  required: ['view:uploadPrivateRecords'],
```

Comment:
Requires selecting either 'yes' or 'no' to the question 'Do you want to upload your Private Medical Records' - assuming they chose PMR on the previous page as a type of supporting evidence they wanted to provide.